### ISSUE TITLE
Typo in Discord 101

### ISSUE THREAD

**Issue by gandalfthegraydelson** [huggingface/deep-rl-class#196]
```
You’ll then get to introduce yourself in the #introduction-yourself channel.
```

It should say #introduce-yourself instead of #introduction-yourself.

**Comment by simoninithomas** [comment]
Thanks for pointing this out I add it to the big weekly update thanks :hugs: . I keep you updated

**Comment by simoninithomas** [comment]
Solved in #230 thanks :hugs:
